cs-minibot-platform-src/src/main/java/simulator/simbot/Dijkstras.java

```diff
@@ -0,0 +1,199 @@
+package simulator.simbot;
+
+import java.util.*;
+
+/**
+ * Created by jimmychen on 4/29/17.
+ */
+public class Dijkstras {
```

Comment:
does this just do Dijkstra's? Or does it also generate mazes, etc? There may be a better class name or a way to break this into multiple classes.